includes/templates/responsive_classic/jscript/jscript_addr_pulldowns.php

```diff
@@ -74,7 +74,7 @@ function update_zone(theForm) {
     theForm.zone_id.options[NumState] = null;
   }
   // build dynamic list of countries/zones for pulldown
-<?php echo zen_js_zone_list('SelectedCountry', 'theForm', 'zone_id'); ?>
+<?php echo zen_js_zone_list('SelectedCountry', 'theForm', 'zone_id', isset($zone_id) ? $zone_id : -1); ?>
```

Comment:
I'm not sure I understand this addition. The `zen_js_zone_list()` function does accept a 4th parameter, but not the one you're passing here.

Reply:
When viewing your address inside the my account address book the zen_js_zone_list doesn't select your state as default, instead defaulting to Please Select.  This patch fixes that.